custom/languages/en_UK.json

```diff
@@ -440,6 +440,7 @@
     "admin/page_url_contains_nameless_path": "Your custom page would overwrite a NamelessMC page.",
     "admin/pages": "Pages",
     "admin/panel_templates": "Panel Templates",
+    "admin/panel_template_third_party": "Panel template {{name}} is a third party template. Please be aware that version compatibility is not guaranteed between minor NamelessMC updates, and issues should be redirected to the template author.",
```

Comment:
This seems really long, open to other ways of wording this!

Reply:
Reads nicely to me 👍